AzureCalling/app/src/main/java/com/azure/samples/communication/calling/external/calling/CallingContext.java

```diff
@@ -190,10 +190,19 @@ public String getJoinId() {
                     final LocalVideoStream[] localVideoStreams = new LocalVideoStream[1];
                     localVideoStreams[0] = localVideoStream;
                     final VideoOptions videoOptions = new VideoOptions(localVideoStreams);
-                    callWithOptions(agent, audioOptions, videoOptions, callLocator);
+                    try {
+                        callWithOptions(agent, audioOptions, videoOptions, callLocator);
+                    } catch (Exception e) {
+                        e.printStackTrace();
+                        throw e;
+                    }
                 });
             } else {
-                callWithOptions(agent, audioOptions, null, callLocator);
+                try {
+                    callWithOptions(agent, audioOptions, null, callLocator);
+                } catch (Exception e) {
+                    throw e;
+                }
```

Comment:
I think these don't need to be in a try/catch either.

Reply:
That's right